src/paths/steiner.cpp

```diff
@@ -0,0 +1,535 @@
+#include "igraph_paths.h"
+#include "igraph_adjlist.h"
+#include "igraph_types.h"
+#include "igraph_memory.h"
+#include "igraph_error.h"
+#include "igraph.h"
+
+#include <cstring>
+#include <cmath>
+#include <map>
+#include <climits>
+#include <vector>
+#include <set>
+#include <iostream>
```

Comment:
`<iostream>`, as well as any terminal output by this function, should be removed before the PR can be merged.

If you use this for debugging, all debug code should be conditional on a macro being defined.